### PR TITLE
Updating README.md

### PR DESCRIPTION
**Pull request by Anubhvv**
v2 components has been added

### Validations

(how the change was tested, including both manual and automated tests)

### Screenshots

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [ ] Automated Tests
- [x] Documentation and demo app examples
- [ ] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and RTL layouts
- [ ] Size classes and window sizes (notched devices, multitasking, different window sizes, etc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +61,56 @@ Some of the controls available include:
...
+- [SearchBarPersonaChip](fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/SearchBarPersonaChip.kt)
+- [Shimmer](fluentui_progress/src/main/java/com/microsoft/fluentui/tokenized/shimmer/Shimmer.kt)
+- [SideRail](fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/navigation/SideRail.kt)
+- [Snackbar](fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/Snackbar.kt)
+- [TabBar](fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/navigation/TabBar.kt)
+- [TabItem](fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt)
+- [TextField](fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/TextField.kt)
+- [ToggleSwitch](fluentui_controls/src/main/java/com/microsoft/fluentui/tokenized/controls/ToggleSwitch.kt)
+- [ToolTip](fluentui_notification/src/main/java/com/microsoft/fluentui/tokenized/notification/ToolTip.kt)
+- [Utils](fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/Utils.kt)
```
Comment (praveen970) [inline README.md:110]: This can be removed as it is not a comtrol

File: README.md [hunk 2]
```diff
@@ -180,23 +231,23 @@ Starting from version 0.0.12, Fluent UI has been split into multiple modules by
...
+|fluentui_calendar         | Calendar, DateTime Picker                                         |                                                                                                                 |
+| fluentui_ccb             |                                                                   | ContextualCommandBar                                                                                            |
+| fluentui_controls        |                                                                   | AnnouncementCard, BasicCard,<br> BasicChip, Button,<br> Checkbox, Citation,<br> FileCard, FloatingActionButton, 
+ <br> Label, RadioButton,<br> TextField, ToggleSwitch |
+| fluentui_core            | TemplateView, Utilities                                           |                                                                                                                 |
+| fluentui_drawer          | Drawer, BottomSheet,<br> Persistent BottomShee                    | BottomSheet, Drawer                                                                                             |
```
Comment (rohanprasadofficial) [inline README.md:241]: nit: BottomSheet

File: README.md (hunk 1 above)
Comment (Anubhvv) [reply README.md]: Done

**Comment by rohanprasadofficial** [comment]
image

as we're updating readme , consider remove this extra brackets on release pipeline badge.

File: README.md
Comment (rohanprasadofficial) [inline README.md:1]: image

as we're updating readme , consider removing this extra brackets on release pipeline badge.

**Review by rohanprasadofficial — CHANGES_REQUESTED** [review]

File: README.md
Comment (Anubhvv) [reply README.md:1]: done

**Review by rohanprasadofficial — APPROVED** [review]
LTGM !

**Review by praveen970 — APPROVED** [review]

**Pull request merged**